Add documentation about OLM installation (#213)
DataDog/datadog-operator: docs/installation.md

   helm install $DD_NAMEOP -n $DD_NAMESPACE .
   ```

### With the Operator Lifecycle Manager

> The Datadog Operator deployment with [Operator Lifecycle Manager][7] documentation is available on the [operatorhub.io][8].

Reviewer: ```suggestion
The Datadog Operator deployment with [Operator Lifecycle Manager][7] documentation is available at [operatorhub.io][8].
```

#### Override default Operator configuration

> The [Operator Lifecycle Manager][7] framework allows overriding default Operator configuration. The list of the supported installation configuration parameters is documented [here][9].

Reviewer: ```suggestion
The [Operator Lifecycle Manager][7] framework allows overriding default Operator configuration. See the [Subscription Config][9] document for a list of the supported installation configuration parameters.
```
